tests/manage/rgw/test_host_node_failure.py

```diff
@@ -1,19 +1,20 @@
 import logging
 import pytest
 
+from ocs_ci.framework.testlib import ManageTest, tier4a, vsphere_platform_required
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.helpers.helpers import wait_for_resource_state
+from ocs_ci.ocs.bucket_utils import s3_put_object, s3_get_object
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.node import get_node_objs
 from ocs_ci.ocs.resources.pod import (
     get_rgw_pods,
     get_pod_node,
     get_noobaa_pods,
     wait_for_storage_pods,
 )
-from ocs_ci.ocs import constants, defaults
-from ocs_ci.ocs.ocp import OCP
-from ocs_ci.framework.testlib import ManageTest, tier4a, vsphere_platform_required
-from ocs_ci.helpers.sanity_helpers import Sanity
-from ocs_ci.helpers.helpers import wait_for_resource_state
-from ocs_ci.ocs.node import get_node_objs
-from ocs_ci.ocs.bucket_utils import s3_put_object, s3_get_object
+from ocs_ci.utility.utils import ceph_health_check
```

Comment:
Should be also skip this on external mode cluster? since rgw is on rhcs
Please allow the test to even run on BM